### ISSUE TITLE
event.getOriginal() == null, from PlayerDestroyItemEvent

### ISSUE THREAD

**Issue by bubbles-mcgee** [MinecraftForge/MinecraftForge#6372]
**Minecraft Version:** 1.14.4

**Forge Version:** 1.14.4-28.1.106

**Logs:** I don't think the log shows anything interesting.

**Steps to Reproduce:**
1. Create an event handler for `PlayerDestroyItemEvent`: 
`public static void itemDestroyed(PlayerDestroyItemEvent event)`
2. Get the item that was destroyed using the getter: 
`ItemStack orig = event.getOriginal();`
3. Check if `orig` is `null`: 
`orig == null`

**Description of issue:**
Among perhaps other conditions, this event is triggered whenever a right click causes the `ItemStack` being used to become empty (see: `processRightClick()` in `PlayerController.class`, event posted by Forge's `onPlayerDestroyItem()`). I may be doing something wrong and I may have misinterpreted the usage of this (correct me if so) but the only case where `event.getOriginal()` returns a non-null is when the item that was 'destroyed' (used up) was in the `MAIN_HAND` and when the item used up is not an `instanceof BlockItem`. 

So basically, it looks like something's going wrong when the `OFF_HAND` is emptied via item breaking or usage (e.g. shield breaks or torch used), and when you use up the last in a stack of blocks (e.g. dirt) in your main hand.

**Comment by DaemonUmbra** [comment]
ItemStacks are never null in 1.14, they are ItemStack.EMPTY
